src/config/Config.ts

```diff
@@ -39,11 +38,15 @@ const conf = {
       linkText: 'Find us on GitHub'
     },
     website: {
-      url: 'http://kiali.io',
+      url: 'https://www.kiali.io', // Without www, we get an SSL error
       icon: 'HomeIcon',
       linkText: 'Visit our web page'
     }
   },
+  /** */
+  documentation: {
+    url: 'https://www.kiali.io/documentation/' // Without www, we get an SSL error
```

Comment:
Can this be overwritten for downstream?

Reply:
I think we can take this value from the env file